pkg/markdown/markdown.go

```diff
@@ -92,6 +94,19 @@ func GetStyle(defaultStyle string) string {
 	return "notty"
 }
 
+func formatImgTags(content string) string {
+	lines := strings.Split(content, "\n")
+	re := regexp.MustCompile(`<img[^>]+\bsrc=["']([^"']+)["']`)
+	for i, line := range lines {
+		images := re.FindAllStringSubmatch(line, -1)
+		for _, element := range images {
+			lines[i] = fmt.Sprintf("![Image](%s)", element[1])
```

Comment:
Instead of the text `Image` there should be the alt text parsed from the `<img>` tag. If the alt text was blank, then this text should be blank as well (`![](SRC)`). This is important for accessibility. A hardcoded alt text that just says `Image` hurts accessibility because it is not useful.